pkg/monitor/intervalcreation/rendering_per_namespace.go

```diff
@@ -93,10 +94,21 @@ type podRendering struct {
 	name string
 }
 
+// ingressServicePodRendering type is used for rendering intervals for services that use the
+// router-default pods found in the openshift-ingress namespace.  This includes image-registry,
+// console, and oauth pods.
+type ingressServicePodRendering struct {
+	name string
+}
+
 func NewPodEventIntervalRenderer() podRendering {
 	return podRendering{}
 }
 
```

Comment:
Functionally it may not matter but for cleanliness and clarity should we set a name for ingressServicePodRendering (& podRendering).  I get we are using the struct to overload the WriteEventData api and the name is probably just to have *something* in the struct but it stood out as I was reviewing that since it is there we could at least set it...

Reply:
interesting observation.  `name` is initialized to empty string -- but `name` is not even used; I guess I could also delete it.